hw/hw_0001/index.2.js

```diff
@@ -1,21 +1,28 @@
-let goods = [
-    {name: "Beer", cost: 1.5},
-    {name: "Milk", cost: 2.5},
-    {name: "Bread", cost: 4.5},
-    {name: "Butter", cost: 1.5},
-    {name: "Sausages", cost: 3},
+let products = [
+  {name: "Beer", number: "0,5L", price: 17.90},
+  {name: "Butter", number: "400g",price: 30.5},
+  {name: "Sausages", number: "400g",price: 50.3},
+  {name: "Eggs", number: "10 pieces", price: 20.55},
+  {name: "Pepsi", number: "2L", price: 23.9},
+  {name: "Bread", number: "400g", price: 12.8},
+  {name: "Kinkali", number: "0.8kg", price: 39.95},
+  {name: "Milk", number: "0.9kg", price: 38.8},
+  {name: "Wheat flour", number: "2kg", price: 42.95},
+  {name: "Condensed milk", number: "300g", price: 79.35},
+  {name: "White chocolate", number: "100g", price: 74.7},
+  {name: "Sugar", number: "1kg", price: 14.5},
+  {name: "Oatmeal", number: "1kg", price: 23.3},
+  {name: "Marshmallows", number: "500g", price: 45.85},
 ];
 
-var date = new Date();
-
 console.log("My buyings:");
 
-goods.forEach((good) => {
-    console.log(`- ${good.name} costs ${good.cost}`);
+products.forEach((product) => {
+  console.log(`- ${product.name} ${product.number} costs ${product.price}`);
```

Comment:
`weight` should be calculated as `product.weight / 1000`
`costs` should be recalculated as a composition of price and quantity `product.price*product.quantity`

Reply:
Ok but modifications have been made to better display with new options
- add parameter `quantity`
- The program now displays the `weight` of the product times the `price` for one product and the `total cost` of the product